pyirf/benchmarks/angular_resolution.py

```diff
@@ -52,8 +56,16 @@ def angular_resolution(
     result[f"{energy_key}_center"] = 0.5 * (energy_bins[:-1] + energy_bins[1:])
     result["n_events"] = 0
 
-    key = "angular_resolution"
-    result[key] = u.Quantity(np.nan, table["theta"].unit)
+    if not isinstance(quantile, Sequence):
+        quantile = [quantile]
+
+    if len(quantile) < 2:
+        keys = ["angular_resolution"]
```

Comment:
I think it's somewhat confusing to change the naming of the output key based on the number of quantiles. Just having a single valuye doesn't mean that that value is what should be defined as "angular resolution" (which is often defined as either 68% containment or 90% containment in different cases).  Wouldn't it be better to just always call them _containment_quantile_XX_ in all cases?  or if you really want to be specific, something like `angular_resolution_{value*100:.0f}pct_containment`.  That way there is no surprise to the user that the column names change.  

I guess the issue is that somewhere else, the code looks for a magic column named "angular_resolution", but then it's also better to make that column name configurable, and have it default to the CTAO standard using the more explicit column naming.

Reply:
Good point, being consistent with the column naming will avoid unnecessary confusion, I'm not sure anymore why I didn't do it like this initially...
Within pyirf only one of the tutorials accesses the column name "angular_resolution", so I'll just change it there accordingly.